### DISCUSSION TITLE
Diskussion zu den Ergebnissen der Analyse

### DISCUSSION THREAD

**Discussion by SchneckenerR** [SchneckenerR/wissenschaftliches-arbeiten#13] [Announcements]
Hey, Svetlana und Lennart sind ja jetzt fertig mit der Analyse und Wir können die Ergebnisse besprechen. Ich würde vorschlagen Lennart und Svetlana heben in der Diskussion bestimmte Ergebnisse hervor und Lena und Ich (Ruth) erklären, wie wir den Datensatz erstellt hatten.

**Comment by 321Svetlana** [comment]
Das Alter der Studenten im Datensatz ist normalverteilt mit Erwartungswert 25 und Standardabweichung 2. Habt ihr das so beabsichtigt beim samplen?

**Reply by smlemons** [reply]
Ihr liegt goldrichtig mit dem Alter! Es ist normalverteilt mit einem Erwartungswert von 25 und einer Standardabweichung von 2

**Comment by 321Svetlana** [comment]
Die Studienfächer sind wie folgt verteilt: 34 % Data Science, 24 % Informatik, 12 % Mathe, 30 % Statistik.

**Reply by smlemons** [reply]
Die Verteilung der Studienfächer ist fast richtig. Data Science und Statistik Studierende habem wir mit einer Wkeit von 3% gesampelt, bei Informatik war es 25% und bei Mathe nur 15%.

**Comment by 321Svetlana** [comment]
Ich vermute, dass die Studenten, die einen Mathe LK hatten, zufällig bestimmt wurden, deshalb ist die Wahrscheinlichkeit 50 %, richtig?

**Reply by smlemons** [reply]
Die Mathe-LK-Variable haben wir mit den gleichen Wkeiten wie beim Matheinteresse für die einzelnen Studiengänge gesampelt, sie ist also nicht studienfachunabhängig zufällig mit p=0.5. Allerdings ergibt der Durchschnitt der Wkeiten, die wir beim sampeln angegebenen haben trotzdem fast 0.5! Ihr habt also indirekt Recht.

**Comment by lkoppe** [comment]
Ok, kommen wir zu den Zusammenhängen der einzelnen Variablen :) 

Wir vermuten, dass es **keinen** Zusammenhang zwischen dem Mathe LK und dem Studienfach gibt. Die Korrelationskoeffizienten sind dafür alle zu gering. Stimmt das?

**Reply by SchneckenerR** [reply]
Doch, zwischen dem Mathe LK und Studienfächern ist eine Korrelation vorhanden, da 60 Prozent der Statistik Studierenden Mathe LK belegt hatten. Bei Mathematik Studierenden hatten 85 Prozent Mathe LK, bei Informatik 40 und Data Science 25 Prozent. Vielleicht sind die Unterschiede einfach zu gering, das war aber unsere eigentliche Intention.

**Comment by lkoppe** [comment]
Dafür gibt es aber mit Sicherheit einen starken Zusammenhang zwischen Mathe LK und dem Interesse an Mathe. Die Korrelationskoeffizienten sind hier sehr hoch.

Gleiches müsste auch für Mathe LK und dem Interesse am Programmieren gelten :)

**Reply by SchneckenerR** [reply]
Richtig! Der prozentuale Anteil, mit dem wir Matheinteresse und  Mathe-LK-Anteil gesampelt haben, ist innerhalb der Studienfächer gleich! Deshalb korreliert Matheinteresse und Mathe-LK-Anteil so stark positiv.

Bei der Korrelation zwischen Mathe LK und Interesse an Programmieren sollte nicht so hoch sein, da wir hier nicht die gleichen Wahrscheinlichkeiten wie auch bei Interesse an Mathematik genommen hatten. Da die Wahrscheinlichkeiten trotzdem ähnlich sind, kann eine leichte Korrelation vorliegen.

**Comment by lkoppe** [comment]
Für das Interesse an Mathe und dem Interesse am Programmieren vermuten wir, dass ein leichter Zusammenhang besteht.

**Reply by SchneckenerR** [reply]
Richtig, da die Interessen beide von der Wahl der Studiengängen abhängen, liegt hier, durch die unterschiedlichen Wahrscheinlichkeiten, nur ein leichter Zusammenhang vor.

**Comment by lkoppe** [comment]
Außerdem meinen wir, dass sehr wahrscheinlich **kein** Zusammenhang zwischen dem Alter und dem Mathe LK besteht.

**Reply by SchneckenerR** [reply]
Genau, diese beiden Variablen sind komplett unabhängig voneinander.
